Autoprefixer.py

```diff
@@ -21,8 +21,8 @@ def get_setting(view, key):
 		settings = sublime.load_settings('Autoprefixer.sublime-settings')
 	return settings.get(key)
 
-def is_css(view):
-	return splitext(basename(view.settings().get('syntax')))[0] == 'CSS'
+def is_style_syntax(view):
+	return splitext(basename(view.settings().get('syntax')))[0] in ['CSS', 'Sass', 'SCSS']
```

Comment:
These refer to syntax packages, and the Sass package also highlights the SCSS syntax.

I'd rather we check the file extension instead. Neither of these syntaxes ships by default, so it doesn't make sense to be aware of them in here.